Todo.elm

```diff
@@ -110,6 +111,14 @@ update action model =
           in
               { model | tasks <- List.map updateTask model.tasks }
 
+      {- UpdateTask updates the description, but we want to know when it was
+      actually submitted, so we can filter out empty descriptions. -}
+      SubmitTask id ->
+          let
+              newModel = update (EditingTask id False) model
+          in
+              { newModel | tasks <- List.filter (\t -> String.length t.description > 0) newModel.tasks }
```

Comment:
:+1: 
I've actually just noted thanks to you comment, the it is possible in the current master to submit a line with empty spaces, so I'll try to take care of it as-well ;)